README.md

```diff
@@ -154,6 +154,64 @@ python -m gcp_variant_transforms.vcf_to_bq \
   --runner DataflowRunner
 ```
 
+### Security and Compliance
```

Comment:
I suggest renaming this as "Running jobs in a particular region/zone" (or something less verbose). I like us to avoid mentioning compliance and we're not 100% sure about how the data is handled across regions.

Reply:
Done!